src/storage/hub.js

```diff
@@ -97,6 +97,15 @@ export function makeV1GaiaAuthToken(hubInfo: Object,
   return `v1:${token}`
 }
 
+/**
+ * 
+ * @param {string} gaiaHubUrl - the Gaia hub URL you want to connect to
+ * @param {string} challengeSignerHex - a private key used for signing an auth token
+ * @param {string} associationToken - an optional association token to add to your auth token
+ * @param {string} scopes - an optional array of scopes to include which will only allow
```

Comment:
We could include better documentation on the `scopes` array here -- basically saying what the scopes array can be used for. What do you think about including that documentation here versus in the gaia docs?

Reply:
I think it'd be best to have them in both places. I'll add more specific notes to the blockstack.js docs here - good idea.